src/OpenTelemetry/Metrics/MeterProviderSdk.cs

```diff
@@ -242,11 +245,19 @@ internal sealed class MeterProviderSdk : MeterProvider
                                 if (metricsSuperList.Any(metrics => metrics.Count > 0))
                                 {
                                     listener.EnableMeasurementEvents(instrument, metricsSuperList);
+                                    enabledMeasurements = true;
                                 }
                             }
                         }
 
-                        OpenTelemetrySdkEventSource.Log.MeterProviderSdkEvent($"Completed publishing Instrument = \"{instrument.Name}\" of Meter = \"{instrument.Meter.Name}\".");
+                        if (enabledMeasurements)
+                        {
+                            OpenTelemetrySdkEventSource.Log.MeterProviderSdkEvent($"Measurements for Instrument = \"{instrument.Name}\" of Meter = \"{instrument.Meter.Name}\" are enabled and would be processed and aggregated by the SDK.");
```

Comment:
I found this a bit hard to understand - what does "measurements are enabled" mean?

Reply:
It means we have called this [MeterListener.EnableMeasurementEvents(Instrument, Object)](https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.metrics.meterlistener.enablemeasurementevents?view=net-8.0) for the given instrument.

Reply:
^ this is an internal thing, how would a user interpret this? (or the log is not intended to be used by the user?)

Reply:
> > ^ this is an internal thing, how would a user interpret this?
> 
> The full message tells what this means for the user: "SDK will not process or it would drop the measurements."
> 
> > or the log is not intended to be used by the user?
> 
> I believe that just because we log some internal things doesn't mean that it cannot be used by users. We could still ask them to collect logs and look for a particular message. Btw we have been logging internal stuff in our logs. See this for [example](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/Metrics/PeriodicExportingMetricReader.cs#L141).

Makes sense, in a nutshell we treat all error/warning logs as user-facing (so they have to be understood by the users, with clear action items), while info/verbose/debug logs are for OpenTelemetry developers (it is okay that the users don't understand the message) - is this aligned with your understanding?

Reply:
Yes, I agree.